src/main/java/de/hs_mannheim/informatik/ct/persistence/repositories/VisitorRepository.java

```diff
@@ -34,10 +34,10 @@
     @Modifying
     @Query("DELETE FROM Visitor visitor " +
             "WHERE visitor NOT IN (" +
-            "   SELECT roomVisit.visitor " +
```

Comment:
Der Whitespace war tatsächlich Absicht um zu zeigen das es zu dem `NOT IN` gehört. Hat IntelliJ das automatisch formatiert?

Reply:
Ne, das war wahrscheinlich ich, habs geändert.